realsense_ros2_camera/src/realsense_camera_node.cpp

```diff
@@ -493,10 +511,6 @@ class RealSenseCameraNode : public rclcpp::Node
             publishFrame(frame, t);
           }
 
-          if (_pointcloud && is_depth_frame_arrived && is_color_frame_arrived) {
```

Comment:
Suggest not modify pointcloud pub in this commit, just add topic pub for aligned_depth_images.